README.md

```diff
@@ -1,10 +1,69 @@
+
+## Basic Segmentation
+
+There is no broad template for building an open source program that applies across all industries, or even across all companies in a single industry. However we can categorize the functions of an OSPO into different categories, depending on the nature of the organization and its motivators.
+
+* `📈 Business-Oriented`: OSPOs established within for-profit organizations, such as enterprises, 
+that are primarily focused on creating business value through the use of open source software. 
+
+    * **Drivers of Motivation:** Innovation / Risk Management / Legal Compliance / Talent Retention
+
+* `🎓 Educational-Oriented`: OSPOs established by educational institutions, such as universities 
+or schools, that are focused on using open source software to support teaching, research and learning 
+activities.
 
-There is no broad template for building an open source program that applies across all industries, or even across all companies in a single industry. However we can categorize the functions of a typical OSPO into three categories: Legal Risk Mitigation, Improving Engineers’ Practices and Enabling Financial Benefits. (A mnemonic: Fear, Love and Money)
+    * **Drivers of Motivation:** Curriculum development / Student Engagement / Open Data / Knowledge Sharing
+
+* `👩‍🏫 Business-Educational`: OSPOs established within for-profit organizations that provide training and certification and are likely to collaborate
+with `educational-oriented` types.
+
+    * **Drivers of Motivation:** hybrid between `📈 Business-Oriented`and `🎓 Educational-Oriented`
+
+
+* `🏛 Social-Gov-Oriented`: OSPOs established within a government or public administrators that are 
+focused on using open source software to achieve social or public policy objectives (e.g serving 
+citizens).
+
+    * **Drivers of Motivation:** Interoperability / Open Data / Accessibility / Inclusion / Privacy / Security / Transparency
+
+* `🌳 Social-NonGov-Oriented`: OSPOs established within non-governmental organizations (NGOs) or foundations that are dedicated 
+to using open source software to create positive social change. 
+
+    * **Drivers of Motivation:** Innovation / Interoperability / Social Justice / Disaster Relief / Humanitarian Aid / Environmental Protection / Sustainable Development
+
+## Drivers of motivation
+
+In this context, motivation is the driving force that fuels the entitiy to achieve its goals and 
+reach its full potential. Based on the categories defined above, we can list some of the functions of an OSPO based on these motivators.
+
+### Business-Oriented Segment
+
+Legal Risk Mitigation, Improving Engineers’ Practices and Enabling Financial Benefits. (A mnemonic: Fear, Love and Money)
```

Comment:
@anajsana, Considering the mnemonic **Fear, Love and Money**... We tend to mitigate not "only legal risks".

What do you think about adding something like Open Source Risks mitigation (Legal, Security and Operational risks)

And just below a very brief description of how we already have Legal Risks for the other two "types of risks"

Reply:
We can just drop the ´legal´ from ´risk mitigation´ so it gives a broader view of the different risks that are, including security, operational, etc 🙂